changelogs/fragments/109-add-timeout-parameter-to-api.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- "api* modules - Added timeout parameter. (https://github.com/ansible-collections/community.routeros/pull/109)."
```

Comment:
```suggestion
- "api* modules - added timeout parameter (https://github.com/ansible-collections/community.routeros/pull/109)."
```

Reply:
I wonder what happened here, GitHub's UI seems to totally screw up the URLs. If I look at the source of my suggestion, the URL is still totally fine... Since you have seemed to simply apply this suggestion, it seems like there's a big bug somewhere in GitHub's suggestion UI...